modules/oadp-ssec-encrypted-backups.adoc

```diff
@@ -0,0 +1,183 @@
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-ssec-encrypted-backups_{context}"]
+= Creating an OADP SSE-C encryption key for additional data security
+
+{aws-first} S3 applies server-side encryption with Amazon S3 managed keys (SSE-S3) as the base level of encryption for every bucket in Amazon S3.
+
+{oadp-first} encrypts data by using SSL/TLS, HTTPS, and the `velero-repo-credentials` secret when transferring the data from a cluster to storage. To protect backup data in case of lost or stolen AWS credentials, apply an additional layer of encryption.
+
+The velero-plugin-for-aws plugin provides several additional encryption methods. You should review its configuration options and consider implementing additional encryption.
+
+You can store your own encryption keys by using server-side encryption with customer-provided keys (SSE-C). This feature provides additional security if your AWS credentials become exposed.
+
+[WARNING]
+====
+Be sure to store cryptographic keys in a secure and safe manner. Encrypted data and backups cannot be recovered if you do not have the encryption key.
+====
+
+.Prerequisites
+
+To use the SSE-C functionality of the `velero-plugin-for-aws` plugin, OADP must mount a secret that contains your SSE-C key.
+
+* To make {oadp-short} mount a secret to the Velero pod at `/credentials`, use the following default secret name for AWS: `cloud-credentials`, and leave at least one of the following labels empty.
+
+** `dpa.spec.backupLocations[].velero.credential`
+** `dpa.spec.snapshotLocations[].velero.credential`
++
+This is a workaround for a known issue: https://issues.redhat.com/browse/OADP-3971.
+
+[NOTE]
+====
+In the following procedure, there is an example of a `spec:backupLocations` block that does not specify credentials. This example would trigger an OADP secret mounting.
+====
+
+* If you need the backup location to have credentials with a different name than `cloud-credentials`, you must add a snapshot location, such as the one in the following example, that does not contain a credential name, meaning the snapshot location will use `cloud-credentials` as its secret for taking snapshots.
+
+.Example snapshot location in a DPA without credentials specified
+[source,yaml]
+----
+ snapshotLocations:
+  - velero:
+      config:
+        profile: default
+        region: <region>
+      provider: aws
+# ...
+----
+
+.Procedure
+. Create an SSE-C encryption key by running the following commands:
+
+.. Generate a random number and output it as `sse.key`.
++
+[source,terminal]
+----
+$ dd if=/dev/urandom bs=1 count=32 > sse.key
+----
+
+.. Encode `sse.key` using Base64 and output the result as `sse_encoded.key`.
```

Comment:
"`sse.key`" -> "the `.sse-key` file" or "the SSE key". 

"using" -> "by using" 

"`sse_encoded.key`" -> "a file `sse_encoded.key`" or "a file named `sse_encoded.key`"

Same for similar language throughout.